packages/connect/src/Detector.test.ts

```diff
@@ -18,4 +17,58 @@ describe('Initialize Detector without extension', () => {
       await detect.disconnect('westend');
     }
   }, 30000);
+
+  test('Should connect with known chain "polkadot".', async done => {
+    try {
+      const chainName = 'polkadot';
+      detect = new Detector('test-uapp');
+      const api = await detect.connect(chainName);
+      expect(api).toBeTruthy();
+      done();
+    } catch (err: unknown) {
+      done(err);
+    } finally {
+      await detect.disconnect('polkadot');
+    }
+  }, 30000);
+
+  // DEACTIVATED for now due to chainSpecs - once Kusama specs are available this should be activated
+  // test('Should connect with known chain "kusama".', async done => {
+  //   try {
+  //     const chainName = 'kusama';
+  //     detect = new Detector('test-uapp');
+  //     const api = await detect.connect(chainName);
+  //     expect(api).toBeTruthy();
+  //     done();
+  //   } catch (err: unknown) {
+  //     done(err);
+  //   } finally {
+  //     await detect.disconnect('kusama');
+  //   }
+  // }, 30000);
+
+  test('Should connect with unknown chain westedn2 and chainSpecs.', async done => {
```

Comment:
Westend typo

Reply:
It does not matter the typo... It can be whatever else than the known networks

Reply:
True but it looks weird 😸